src/modules/components/dao/daoDataListItemStructure/daoDataListItemStructure.tsx

```diff
@@ -0,0 +1,48 @@
+import type React from 'react';
+import { DataList, Heading, Icon, IconType, type IDataListItemProps } from '../../../../core';
+import { DaoAvatar } from '../daoAvatar';
+
+export interface IDaoDataListItemStructureProps extends IDataListItemProps {
+    name?: string;
+    logoSrc?: string;
+    description?: string;
+    address?: string;
+    ens?: string;
+    plugin?: string;
+    network?: string;
+}
+
+export const DaoDataListItemStructure: React.FC<IDaoDataListItemStructureProps> = (props) => {
+    const { name, logoSrc, description, network, plugin = 'token-based', address, ens, ...otherProps } = props;
+
+    return (
+        <DataList.Item {...otherProps}>
```

Comment:
Looks like we need to increase the y padding on the dao-data-list-item component, on the design it's greater